api/indigo.h

```diff
@@ -208,7 +212,7 @@ CEXPORT const char* indigoCdxml(int object);
 // (because MDLCT data usually contains zeroes)
 CEXPORT int indigoSaveMDLCT(int item, int output);
 
-CEXPORT const char* indigoJson(int object);
+// CEXPORT const char* indigoJson(int object);
```

Comment:
Better remove this function completely along with its implementation.

Reply:
fixed